src/index.vue

```diff
@@ -88,6 +88,12 @@ export default {
     .on('circle-inited', function(event){
       renderCircleBody(this, (vm.progress/vm.scale));
       vm.$emit('vue-circle-init', event);
+      vm.$watch('progress', (nv) => {
```

Comment:
Can we put these watcher methods into the "watched" property?
```
 watch: {
    fill(nv) {
      this.updateFill(nv);
    },
    progress(nv) {
      this.updateProgress(nv);
    },
  },
```
That will help with keep the functions more intentional.